ci/codebuild_start_wait.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -eu -o pipefail
+
+[[ $# -lt 2 ]] && {
+    echo "Usage $0 <PROJECT_NAME> <S3_URL>" >&2
+    echo "  Example: $0 project-name s3://mybucket/path" >&2
+    exit 1
+}
+
+PROJECT_NAME=$1
+S3_URL=$2
+
+CB_ENV_OVERRIDE='[{"name":"TF_VAR_scan_s3_url","value":"'"$S3_URL"'","type":"PLAINTEXT"}]'
+
+WAIT_INTERVAL=30 #in seconds
+
+if [ -z $(aws codebuild list-projects  --output text --query "projects[? @ == '${PROJECT_NAME}']") ]; then
+  echo "Codebuild command failed or project not found!"
+  exit 1
+else
+  echo "Codebuild project found!  Starting build job..."
+  BUILD_ID=$(aws codebuild start-build --project-name ${PROJECT_NAME} --environment-variables-override ${CB_ENV_OVERRIDE} --output text --query 'build.id')
+fi
+
+build_status() {
+  aws codebuild batch-get-builds --ids ${BUILD_ID} --query 'builds[*].buildStatus' --output text
+}
+
+echo "Start checking status for build..."
```

Comment:
Can we construct the console url for the build, so we can copy and go? Echo it out at the start here, and also at the end?